README.md

```diff
@@ -1,20 +1,44 @@
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+A private beta service for:
```

Comment:
Time will fly by, and before we know it we will be out of private beta into public and then live.  With that in mind, I would rephrase this to something like: The frontend application enables: